.github/actions/next-stats-action/src/constants.js

```diff
@@ -5,8 +5,8 @@ const benchTitle = 'Page Load Tests'
 const workDir = path.join(os.tmpdir(), 'next-stats')
 const mainRepoName = 'main-repo'
```

Comment:
Where else is `mainRepoName` and `diffRepoName ` used? Could we delete those variables?

Reply:
This is just a revert of different PR, but sure I can improve this.